config/sync/block.block.usertimezoneblock.yml

```diff
@@ -0,0 +1,20 @@
+uuid: 79adec60-f230-402c-a160-e0e421b15b2f
+langcode: en
+status: true
+dependencies:
+  module:
+    - user_timezone
+  theme:
+    - subtheme_videogame
+id: usertimezoneblock
+theme: subtheme_videogame
+region: navbar_right
```

Comment:
Is this the final placement of the block? As I remember it should be in the header according to the front page design plan.